Documentation/operations/upgrade.rst

```diff
@@ -307,7 +307,10 @@ Annotations:
 1.16 Upgrade Notes
 ------------------
 
-* To be determined during v1.16 development.
+* Cilium Envoy daemon set is now enabled by default. If you are upgrading from a
+  previous version, you need to enable it manually by setting ``envoy.enabled=false``
```

Comment:
Shouldn't this be 
```suggestion
  previous version, you need to disable it manually by setting ``envoy.enabled=false``
```

And maybe mention "if you want to preserve existing behavior"?

Reply:
ah right, thanks a lot, let me push this change 👍